test/integration.spec.js

```diff
@@ -68,3 +70,20 @@ test('integration : babel syntax error', t => {
     ''
   ]);
 });
+
+test('integration : webpack multi compiler', t => {
+
+  const syncWebpackWithPlugin = deasync(function(config, fn) {
+    const compiler = webpack(config);
+    // Plugin must be applied to `MultiCompiler` rather than via webpack.config.
+    compiler.apply(new FriendlyErrorsWebpackPlugin());
+    compiler.run(fn);
+    return compiler;
+  });
+
+  var logs = output.captureLogs(() => {
+    syncWebpackWithPlugin(require('./fixtures/multi-compiler/webpack.config'));
+  });
+
+  assert(/DONE  Compiled successfully in /.test(logs));
```

Comment:
Good point, it wouldn't have actually failed on a successful build; I guess I was thinking more about future updates -- I've made some changes now which means it would have failed :)